### DISCUSSION TITLE
Seperate option for Beatmap Combo Color

### DISCUSSION THREAD

**Discussion by Fisky1** [ppy/osu#13095] [Ideas]
Very simple option to add to the game. A separate option to enable/disable the beatmap combo color settings. This is especially useful for mappers so they don't need to enable and disable beatmap skin every time they would like to play the game instead of mapping.

**Comment by bdach** [comment]
To clarify, you would like a global override for disabling combo colours?

This feels iffy because if you enable combo colours globally and keep the beatmap skin setting then you also need to have beatmap skins not disabled to actually get the proper colours, but the disable will work everywhere on its own. This'll be confusing to users I'd imagine.

**Comment by Fisky1** [comment]
No, that is not what I mean. If you have an option to disable Beatmap CC then the option for beatmap skin should not have anything with CC to do.

**Comment by MightyHelper** [comment]
What I understood from this is :
Ignore the default CC in the (current) skin.ini and only look at the map [[Colours]](https://osu.ppy.sh/help/wiki/osu!_File_Formats/Osu_(file_format)) section?
I thought the Colours section overwrote the current skin CC?

**Comment by Fisky1** [comment]
It does overwrite currently, but only if you enable beatmap skin. I do not think most players want to play with an old beatmap skin on, for example, a 2009 map.

**Comment by un2wo** [comment]
I was gonna suggest something like this, so I'll add something here instead of making a new post (which may or may not be clarifying OP's words, I'm not sure)

Basically, a way to preserve beatmap combo colors when the "beatmap skin" option is disabled would be nice.
For example, some people (such as me) like having separate combo colors per beatmap for variety, but are bothered when there's custom hitcircles or different slidertrack/sliderborder colors that can conflict with the user's skin.
It'd be similar to going into a beatmap's folder and manually deleting the skin elements on stable.
